SIL.Core/Reporting/ErrorReport.cs

```diff
@@ -138,62 +138,29 @@ public static void Init(string emailAddress)
 			ErrorReport.AddStandardProperties();
 		}
 
-		/// ------------------------------------------------------------------------------------
-		/// <summary>
-		///
-		/// </summary>
-		/// <param name="error"></param>
-		/// <returns></returns>
-		/// ------------------------------------------------------------------------------------
-		public static string GetExceptionText(Exception error)
+		private static void UpdateEmailSubject(Exception error)
 		{
 			var subject = new StringBuilder();
 			subject.AppendFormat("Exception: {0}", error.Message);
-
-			var txt = new StringBuilder();
-
-			txt.Append("Msg: ");
-			txt.AppendLine(error.Message);
-
-			try
-			{
-				var cOMException = error as COMException;
-				if (cOMException != null)
-				{
-					txt.Append("COM message: ");
-					txt.AppendLine(new Win32Exception(cOMException.ErrorCode).Message);
-				}
-			}
-			catch {}
-
 			try
 			{
-				txt.Append("Source: ");
-				txt.AppendLine(error.Source);
 				subject.AppendFormat(" in {0}", error.Source);
 			}
 			catch {}
-
-			try
-			{
-				if (error.TargetSite != null)
-				{
-					txt.Append("Assembly: ");
-					txt.AppendLine(error.TargetSite.DeclaringType.Assembly.FullName);
-				}
-			}
-			catch {}
-
-			try
-			{
-				txt.Append("Stack: ");
-				txt.AppendLine(error.StackTrace);
-			}
-			catch {}
-
 			s_emailSubject = subject.ToString();
+		}
 
-			return txt.ToString();
+		/// ------------------------------------------------------------------------------------
+		/// <summary>
+		///
+		/// </summary>
+		/// <param name="error"></param>
+		/// <returns></returns>
+		/// ------------------------------------------------------------------------------------
+		public static string GetExceptionText(Exception error)
+		{
+			UpdateEmailSubject(error);
```

Comment:
This is an unexpected side-effect of a "Get" method. But it was there before this PR... it's just less hidden now :-)